homeassistant/components/samsungtv/bridge.py

```diff
@@ -522,7 +522,7 @@ async def async_try_connect(self) -> str:
                 return RESULT_AUTH_MISSING
             except (ConnectionFailure, OSError, AsyncioTimeoutError) as err:
                 LOGGER.debug("Failing config: %s, %s error: %s", config, type(err), err)
-        # pylint: disable=useless-else-on-loop
+        # pylint: disable-next=useless-else-on-loop
         else:
             if result:
                 return result
```

Comment:
There is no `break` in the `for` loop, so the else will be executed every time.
It would be a small code change but the `else` can be removed and the code de-dented.

Reply:
Sorry, I want to keep code changes out of scope for this PR.